addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -15,6 +15,17 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
             // Force to rerender the FC.
             // As it doesn't redraw the header when the event's data changes
             this.fc.api.render();
+            /*The fc.api.rerender() only re-renders content handled by its internal virtual DOM
+            and content added using methods like dayCellContent or eventContent
+            (https://fullcalendar.io/docs/content-injection)
+
+            Since the dayCell content is injected into the DOM using standard js,
+            the lib doesn't know about it and won't rerender it
+
+            From the docs (https://fullcalendar.io/docs/Calendar-render):
+            'This method will not completely wipe the DOM clean and rebuild. It will use its
+            internal virtual DOM representation to only commit needed changes.'*/
+            this.rerenderDayCells();
```

Comment:
I'm a bit confused because we already have all these rendering hooks like `onDayHeaderDidMount` or `onEventDidMount`, why didn't we use those?

I had a look in fullcalendar and it seems probably ok technically as we don't really use any of the asynchronous features of the lib. But seems like we should stick to one way of doing things, ideally. Unless it's not compatible with what we're doing.